app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,35 @@
+.required-loan-types
+  label = t('loan_questions.required_loan_types')
```

Comment:
I believe our styleguide calls for double quotes in Rails/Ruby code. I know this project is a mishmash but I'd like to adhere to our styleguide anyway.